sql-cli/sql_cli/project.py

```diff
@@ -168,11 +206,18 @@ def load_config(self, environment: str = DEFAULT_ENVIRONMENT) -> None:
         config = Config(environment=environment, project_dir=self.directory).from_yaml_to_config()
         if config.airflow_home:
             self._airflow_home = Path(config.airflow_home).resolve()
+        else:
+            self._initialise_env_config(environment)
+            config = Config(environment=environment, project_dir=self.directory).from_yaml_to_config()
+
+        if not self.get_env_airflow_home(environment).exists():
+            self._initialise_airflow_for_environment(environment)
+
         if config.airflow_dags_folder:
             self._airflow_dags_folder = Path(config.airflow_dags_folder).resolve()
         if config.data_dir:
             self._data_dir = Path(config.data_dir).resolve()
-        reload_airflow(self.airflow_home)
+        reload_airflow(self.get_env_airflow_home(environment))
```

Comment:
```suggestion
        reload_airflow(self._airflow_home)
```
?

Reply:
They are not the same thing, `self._airflow_home` is used during setup if the user explicitly specifies it